legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-snowflake/legend-engine-xt-relationalStore-snowflake-execution/pom.xml

```diff
@@ -188,6 +188,11 @@
             <artifactId>legend-engine-pure-code-compiled-core</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-executionPlan</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
```

Comment:
Can we confirm this setup wont end running test cases defined in this module twice?

Reply:
They run only once. We need it to reuse AbstractTestSemiStructured without having to duplicate it in Snowflake module (in general each db extension that supports semi-structured data)